components/dashboard/src/settings/Integrations.tsx

```diff
@@ -580,10 +597,10 @@ function GitIntegrationModal(props: ({
         }
     };
 
-    return (<Modal visible={!!props} onClose={close}>
-        <h3 className="pb-2">{props.mode === "new" ? "New Git Integration" : "Git Integration"}</h3>
+    return (<Modal visible={!!props} onClose={close} closeable={props.closeable}>
+        <h3 className="pb-2">{mode === "new" ? "New Git Integration" : "Git Integration"}</h3>
         <div className="space-y-4 border-t border-b border-gray-200 dark:border-gray-800 mt-2 -mx-6 px-6 py-4">
```

Comment:
**issue**: Just tried this in a smaller screen and realized we're going to need some scrolling overflow behavior to fit the modal. Currently, this slightly crops the modal on the bottom but should be ok for this iteration. 

| Modal Crop |
|-|
| ![image](https://user-images.githubusercontent.com/120486/115240756-be653680-a128-11eb-905f-1847af0e8761.png) |